thunder.info.yml

```diff
@@ -9,6 +9,12 @@ distribution:
 
 # Required modules
 dependencies:
+  - config_selector
+  - media
+  - node
+  - views
```

Comment:
I am unsure about the views dependency. It is possible to do meaningful sites without views and we just need them for the content list, which would fall back to the entity list without views.

Reply:
This PR is for replacing the existing behavior with core functionality. For changing the behavior we should create a new PR and define which dependencies are useful